### PR TITLE
Pull both the base and head shas from AMIs when message PRs, and mess…

### PR DESCRIPTION
**Pull request by cpennington**
…age both edx-platform and edx-platform-private

### CODE REVIEW

File: edxpipelines/patterns/tasks/common.py [hunk 1]
```diff
@@ -1589,42 +1592,66 @@ def generate_message_pull_requests_in_commit_range(
...
-        '--head_sha', head_sha,
-        '--{}'.format(flag_for_release_status[release_status])
+        '--release', flag_for_release_status[release_status],
     ]
     if base_sha:
-        arguments.extend(['--base_sha', base_sha])
+        arguments.extend(['--base-sha', base_sha])
+    else:
+        if not (base_ami_artifact and base_ami_tag_app):
+            raise ValueError("base_ami_artifact and base_ami_tag_app must be specified in base_sha isn't")
```
Comment (doctoryes) [inline edxpipelines/patterns/tasks/common.py]: in -> if

File: edxpipelines/patterns/tasks/common.py [hunk 2]
```diff
@@ -1589,42 +1592,66 @@ def generate_message_pull_requests_in_commit_range(
...
-            '--ami_tag_app', ami_tag_app,
+            '--base-ami-tags', "../{}/{}".format(constants.ARTIFACT_PATH, base_ami_artifact.file_name),
+            '--base-ami-tag-app', base_ami_tag_app,
+        ])
+
+    if head_sha:
+        arguments.extend(['--head-sha', head_sha])
+    else:
+        if not (head_ami_artifact and head_ami_tag_app):
+            raise ValueError("head_ami_artifact and head_ami_tag_app must be specified in head_sha isn't")
```
Comment (doctoryes) [inline edxpipelines/patterns/tasks/common.py]: in -> if

File: edxpipelines/patterns/tasks/common.py [hunk 3]
```diff
@@ -1589,42 +1592,66 @@ def generate_message_pull_requests_in_commit_range(
...
     }
 
     arguments = [
         '--org', org,
-        '--token', token,
+        '--token', '${GITHUB_TOKEN}',
         '--repo', repo,
-        '--head_sha', head_sha,
-        '--{}'.format(flag_for_release_status[release_status])
+        '--release', flag_for_release_status[release_status],
```
Comment (doctoryes) [inline edxpipelines/patterns/tasks/common.py]: Won't this change require a tubular update?

Comment (cpennington) [inline edxpipelines/patterns/tasks/common.py]: https://github.com/edx/tubular/pull/171/files#diff-c78f353245a975f05f340fc5938dc2c0R76

Comment (doctoryes) [inline edxpipelines/patterns/tasks/common.py]: Ah - hard to keep track 🚤

**Pull request merged**